openff/interchange/interop/gromacs/export/_export.py

```diff
@@ -456,7 +457,8 @@ def _write_gro(self, gro, decimal: int):
                         f"%{decimal + 5}.{decimal}f"
                         f"%{decimal + 5}.{decimal}f\n"
                         % (
-                            atom.residue_index,  # This needs to be looked up from a different data structure
+                            residue_id = 1,
+                            # (hidden: 1013) atom.residue_index,  # This needs to be looked up from a different data structure
```

Comment:
Could you help me understand why the residue number should be hardcoded instead of looked up from the topology? Sometimes residue information is meaningless but it's often useful, and modifying it at export time is something I'd like to avoid